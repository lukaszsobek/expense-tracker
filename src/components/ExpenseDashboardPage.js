import React from "react"

import { ExpenseList, ExpenseListFilters } from "./"

const ExpenseDashboardPage = () => (
    <div>
        <ExpenseListFilters />
        <ExpenseList />
    </div>
)

export default ExpenseDashboardPage