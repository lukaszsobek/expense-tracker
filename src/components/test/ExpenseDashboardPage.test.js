import React from "react"
import { shallow } from "enzyme"

import { ExpenseDashboardPage } from "../ExpenseDashboardPage"

test("Renders ExpenseDashboardPage", () => {
    const wrapper = shallow(<ExpenseDashboardPage />)
    expect(wrapper).toMatchSnapshot()
})