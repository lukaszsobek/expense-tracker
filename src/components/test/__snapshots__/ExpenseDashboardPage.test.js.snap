// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders ExpenseDashboardPage 1`] = `
<div
  className="app-body"
>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList)
    expenses={Array []}
  />
</div>
`;
